src/aggregator/aggregator/placement_mgr.go

```diff
@@ -122,6 +125,13 @@ func (mgr *placementManager) Open() error {
 	return nil
 }
 
+func (mgr *placementManager) InstanceID() string {
```

Comment:
nit: may need a comment here

Reply:
As this is an unexported type (`placementManager`) it's not required thankfully (only comments on the exported interface definitions is required).